requirements/dev.txt

```diff
@@ -8,8 +8,7 @@ pytest-cov==2.10.1
 # mypy adds new rules in new minor versions, which could cause our PR check to fail
 # here we fix its version and upgrade it manually in the future
 mypy==0.790
-# 1.17.91 has a bug, https://github.com/vemel/mypy_boto3_builder/pull/82
-boto3-stubs[essential]==1.17.90.post1
+boto3-stubs[essential]~=1.18
```

Comment:
I also updated this one. Let me know if this should be different from `base.txt`